src/web/routes/application/steps/address/postcode/validate.js

```diff
@@ -3,13 +3,19 @@ const { check } = require('express-validator')
 const { translateValidationMessage } = require('../../common/translate-validation-message')
 const { UK_POSTCODE_PATTERN, CHANNEL_ISLANDS_AND_IOM_POSTCODE_PATTERN } = require('../constants')
 
+const checkPostcodeIsValid = (_, { req }) => UK_POSTCODE_PATTERN.test(req.body.sanitizedPostcode)
```

Comment:
I don't understand this notation `(_` what does that mean?

Reply:
It's a variable that we don't care about.